src/lib/samples/curl.ts

```diff
@@ -0,0 +1,30 @@
+import type { CodeSampleContext, CodeSampleDefinition } from './code-sample.js'
+import { createJsonResponse } from './json.js'
+
+const BASE_URL = 'https://connect.getseam.com'
+
+export const createCurlRequest = (
+  { request }: CodeSampleDefinition,
+  _context: CodeSampleContext,
+): string => {
+  const method = 'POST'
```

Comment:
I use POST in all samples as we do it in SDKs. But let me know if we should use `_context.endpoint.request.semanticMethod` or `preferredMethod`